src/v2/Apps/getAppNovoRoutes.tsx

```diff
@@ -2,21 +2,24 @@ import { buildAppRoutes } from "v2/Artsy/Router/buildAppRoutes"
 import { RouteConfig } from "found"
 import { routes as artistRoutes } from "v2/Apps/Artist/routes"
 import { debugRoutes } from "./Debug/debugRoutes"
-import { cloneDeep } from "lodash"
 
 export function getAppNovoRoutes(): RouteConfig[] {
   return buildAppRoutes([
     {
-      routes: cloneDeep(artistRoutes).map(route => {
```

Comment:
Also this, since we're only updating path one level deep during initial haul we can just spread things into a new object and avoid the cloneDeep completely. Clicked around and things seemed ok but I might be missing something somewhere.